test/end-to-end/pages/deceased/foreigndeathcerttranslation.js

```diff
@@ -1,14 +1,21 @@
 'use strict';
 
 const config = require('config');
-const commonContent = require('app/resources/en/translation/common');
-const content = require('app/resources/en/translation/deceased/foreigndeathcerttranslation');
+const commonContentEn = require('app/resources/en/translation/common');
+const commonContentCy = require('app/resources/cy/translation/common');
+const contentEn = require('app/resources/en/translation/deceased/foreigndeathcerttranslation');
+const contentCy = require('app/resources/cy/translation/deceased/foreigndeathcerttranslation');
 
-module.exports = async function(answer) {
+module.exports = async function(language = 'en', answer) {
     const I = this;
+    const commonContent = language === 'en' ? commonContentEn : commonContentCy;
+    const foreignDeathCertTranslationContent = language === 'en' ? contentEn : contentCy;
 
     await I.checkPageUrl('app/steps/ui/deceased/foreigndeathcerttranslation');
-    await I.waitForText(content.question, config.TestWaitForTextToAppear);
+    if (language === 'en') {
+        await I.waitForText(foreignDeathCertTranslationContent.question, config.TestWaitForTextToAppear);
+    }
```

Comment:
again, not sure why you'd only want to check the text here on English journeys? It will likely help with test stability for both languages